.github/stale.yml

```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 60
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+staleLabel: wontfix
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+closeComment: false
+only: issues
```

Comment:
Title says issues and PRs, but this seems to be targeting issues only. Was this intentional?

Reply:
Fixed.